_config.yml

```diff
@@ -33,3 +34,8 @@ exclude:
   - cache
 collections:
   hackers:
+what_we_do: >
+  We are a self managed group that work all together and  believe into share
+  knowled to improve and grow.
+  we don't charge people that want to work in this space with us,
```

Comment:
we don't charge anything to people that want to work in this space with us but we expect they wish the same.